test/lib/pgBackRestTest/Module/Backup/BackupFileUnitPerlTest.pm

```diff
@@ -212,26 +210,20 @@ sub run
         # Backup.manifest not written but backup.manifest.copy written because size threshold  met
         $self->testResult(sub {storageTest()->exists("$strBackupPath/" . FILE_MANIFEST . INI_COPY_EXT)}, true,
             'backup.manifest.copy exists in repo');
-        $self->testException(sub {storageRepo()->openRead("$strBackupPath/" . FILE_MANIFEST)}, ERROR_FILE_MISSING,
-            "unable to open '$strBackupPath/" . FILE_MANIFEST . "': No such file or directory");
+        $self->testResult(
+            sub {storageRepo()->exists("$strBackupPath/" . FILE_MANIFEST)}, false, 'backup.manifest.copy missing in repo');
 
         storageTest()->remove($strFileRepo);
         storageTest()->remove($strPgControlRepo);
 
         #---------------------------------------------------------------------------------------------------------------------------
-        # No prior checksum, yes compression, yes page checksum, no extra, no delta, no hasReference
-        $self->testException(sub {backupFile($strFileDb, $strRepoFile, $lFileSize, undef, true,
-            $strBackupLabel, true, cfgOption(CFGOPT_COMPRESS_LEVEL), $lFileTime, true, undef, false, false, undef)}, ERROR_ASSERT,
-            "iWalId is required in Backup::Filter::PageChecksum->new");
```

Comment:
I don't understand why this test was removed.

Reply:
Because in C this parameter is passed as an unsigned int so it is not optional, therefore you don't need to check if it was passed.  Anyway we have full unit tests for page checksum now so this test would be redundant in any case.